runtime/middleware/jwt.go

```diff
@@ -205,34 +205,41 @@ func jwksEndpointsJsonToKeys(ctx context.Context, jwksEndpointsJson string) (map
 		return nil, err
 	}
 	keys := make(map[string]any)
-	for key, value := range jwksEndpoints {
+	for endpointKey, value := range jwksEndpoints {
 		jwks, err := jwk.Fetch(ctx, value)
 		if err != nil {
 			return nil, err
 		}
 
-		jwkKey, exists := jwks.Key(0)
-		if !exists {
-			return nil, errors.New("No keys found in JWKS for key: " + key)
+		if jwks.Len() == 0 {
+			return nil, errors.New("No keys found in JWKS for endpoint: " + endpointKey)
```

Comment:
I think we can remove this test and just let the loop continue.  If no keys are found, then the result will be empty, and that will be handled by the caller already.